src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -3,28 +3,17 @@ title: JavaScript
 sidebar_order: 3
 ---
 
-{% include learn-sdk.md platform="javascript" %}
 
-All our JavaScript-related SDKs provide the same API still there are some differences between them which this section of the docs explains.
-
-## Browser Compatibility
-
-Our JavaScript SDK supports all major browsers. In older browsers, error reports may have a degraded level of detail – for example, missing stack trace data or missing source code column numbers.
-
-The table below shows supported browsers:
-
-![Sauce Test Status](https://saucelabs.com/browser-matrix/sentryio.svg)
 
 {% capture __alert %}
```

Comment:
Weird, that "{% capture __alert %}" on line 8 makes everything bold until "___PUBLIC_KEY___" 700+ lines later in GitHub.

Reply:
Markdown parser sees `__` so it's accurate behavior. There's no `{% %}` in markdown spec, thus it's ignored. It'll work fine once it's built.

Reply:
Great! Thanks, @kamilogorek!